pom.xml

```diff
@@ -120,6 +121,17 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
it is used internally by cxf, in SpringBusFactory. It references:
```import org.springframework.beans.BeansException;
import org.springframework.beans.factory.xml.NamespaceHandlerResolver;
import org.springframework.context.ApplicationContext;
import org.springframework.context.ConfigurableApplicationContext;
import org.springframework.core.io.Resource; ```

I don't think we might be able to remove these. They are internal, anyway.